tests/test_h3_failed_set_atomicity.py

```
@@ -0,0 +1,32 @@
+        max_bucket_size=1,
+    )
+
+    await hamt.set("a", b"value-a")
+    assert await hamt.get("a") == b"value-a"
+
+    with pytest.raises(IndexError):
+        await hamt.set("b", b"value-b")
+
+    try:
+        committed_value = await hamt.get("a")
+    except Exception as error:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)